src/modes/direct_select.js

```diff
@@ -158,6 +166,7 @@ module.exports = function(ctx, opts) {
     },
     stop: function() {
       doubleClickZoom.enable(ctx);
+      ctx.store.clearSelectedCoordinates();
```

Comment:
When deselecting a point by clicking on the feature in direct_select mode the draw.selectionchange does not emit. This action keeps the user in direct_select. If I click off of the feature (which moves me to simple_select) the event is emitted.

It should be emitted both times.